.changeset/afraid-donuts-ring.md

```diff
@@ -0,0 +1,6 @@
+---
+"@mysten/wallet-standard": minor
+"@mysten/sui.js": minor
+---
+
+Change `signMessage` to return message bytes. Add support for sui:signMessage in the wallet standard
```

Comment:
ditto, signPersonalMessage

Reply:
`sui:signMessage` naming is designed to align with wallet standards from other chains, so would like to keep that the same. We could change the internal method names but then they don't align across our SDK, which seems non-ideal.

Reply:
Yeah I get the desire to keep alignment with rust, although we abstract over that by adding the intent automatically, so the `PersonalMessage` intent is effectively invisible to consumers of these APIs.

Reply:
yes that's fine, I think a small doc comment could slightly help here, but overall we're good with the PR